src/gsy_e/models/area/scm_manager.py

```diff
@@ -299,7 +300,15 @@ def community_bills(self) -> Dict:
             community_bills.import_grid_fees += data.import_grid_fees
             community_bills.export_grid_fees += data.export_grid_fees
 
-        return community_bills.to_dict()
+            savings_from_buy_from_community += data.savings_from_buy_from_community
+            savings_from_sell_to_community += data.savings_from_sell_to_community
+            savings += data.savings
+
+        comm_bills = community_bills.to_dict()
+        comm_bills["savings"] = savings
+        comm_bills["savings_from_buy_from_community"] = savings_from_buy_from_community
+        comm_bills["savings_from_sell_to_community"] = savings_from_sell_to_community
```

Comment:
So you decided to export all of them, even though the "savings" already include the other two for the "normal" AreaEnergyBills ?
I guess this is in order not to break the existing workflow, right? However, I do not see the last two begin used on the gsy-web. Can you please point me to them?

Reply:
I was actually thinking to save these in the DB for debugging purposes, but later on I figured out that it is probably an overkill to maintain them just for debugging. Will remove these both from here and from the gsy-framework, thanks for catching this!